homeassistant/components/sms/notify.py

```diff
@@ -12,30 +12,28 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We can remove the whole schema as the config won't be used. The platform setup from config will fail as we return `None` below for the notification service.